src/components/table/DataTableProperties.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/macro'
+
+import { RESOURCES } from '../entry/resources'
+
+const TableLinkPreview = styled.img`
+  width: 20px;
+  height: 20px;
+  margin-right: 5px;
+`
+
+const customLinkSort = (rowOne, rowTwo) => {
+  if (rowOne.links.length > rowTwo.links.length) {
+    return 1
+  } else if (rowOne.links.length < rowTwo.links.length) {
+    return -1
+  }
+  return 0
+}
+
+const ResourceList = ({ url, key }) =>
+  RESOURCES.map(
+    ({ title, urlKey, icon }) =>
+      url.includes(urlKey) && (
+        <TableLinkPreview src={icon} key={key} alt={`${title} logo`} />
+      ),
+  )
+
+let FORMATTERS = {
```

Comment:
We can use const here :)

Reply:
sorry  i changed it to let because I had some weird import issues and thought that made a difference for whatever reason